Makefile

```diff
@@ -1,3 +1,7 @@
+ifneq (3.82, $(firstword $(sort $(MAKE_VERSION) 3.82)))
```

Comment:
I'm just noting here that this is a lexical sort ([ref](https://www.gnu.org/software/make/manual/html_node/Text-Functions.html#index-sorting-words))

This would eventually break with Make version 10, which is not a near-term concern.

Reply:
@silenceshell could you try to compute with `bc` or something like this:

https://stackoverflow.com/a/21491594